webapi/src/main/scala/org/knora/webapi/store/eventstore/EventStore.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright © 2015-2021 the contributors (see Contributors.md).
+ *
+ *  This file is part of the DaSCH Service Platform.
+ *
+ *  The DaSCH Service Platform  is free software: you can redistribute it
+ *  and/or modify it under the terms of the GNU Affero General Public
+ *  License as published by the Free Software Foundation, either version 3
+ *  of the License, or (at your option) any later version.
+ *
+ *  The DaSCH Service Platform is distributed in the hope that it will be
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+ *  of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public
+ *  License along with the DaSCH Service Platform.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ */
+
+package org.knora.webapi.store.eventstore
+
+import akka.http.scaladsl.util.FastFuture
+import com.eventstore.dbclient._
+import org.knora.webapi.IRI
+import zio.{BootstrapRuntime, IO, Task}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait EventStore {
+  def saveResourceEvent(event: ResourceEvent): Future[Boolean]
+  def loadResourceEvents(resourceIri: IRI)(implicit ec: ExecutionContext): Future[List[ResourceEvent]]
```

Comment:
Does `loadResourceEvents` load all events since the beginning of time? Is there a way to subscribe to events?